client/overview/index.js

```diff
@@ -22,8 +26,20 @@ const OverviewPage = () => {
 	} = wcpaySettings;
 
 	const tasks = getTasks( { accountStatus, showUpdateDetailsTask } );
+	const searchQuery = parse( window?.location?.search );
```

Comment:
Why is optional chaining required here? Even in tests `window.location` is present.

Reply:
In case we were using server-side rendering, but we don't (just checked). But it is okay to keep it around in case we migrate, it won't break. What do you think?

Reply:
Let's rely on `window.location` availability and omit the `?.` constructs.

Reply:
Fixed it in: https://github.com/Automattic/woocommerce-payments/pull/1895/commits/65d094196259ae9d7c2dff3deabfc05fc0cdd55d

Reply:
@naman03malhotra have you considered using `getQuery` from `@woocommerce/navigation` package instead of qs? [Here is usage example](https://github.com/Automattic/woocommerce-payments/blob/65d094196259ae9d7c2dff3deabfc05fc0cdd55d/client/transactions/list/index.js#L141-L141).